From 47cd22cd213d2c5f28e674b12c264dc731682560 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Tue, 5 Jun 2018 14:58:42 +0300
Subject: [PATCH] Somehow this module breaks through the perl wrapper and
 declares perl binary to be 'perl.real'. This patch forces it back to perl.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Unix.pm | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Unix.pm b/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Unix.pm
index fe38598..0828f52 100644
--- a/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Unix.pm
+++ b/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Unix.pm
@@ -1161,6 +1161,9 @@ WARNING
     }
 
     foreach my $name (@$names){
+        # Getting MakeMaker.pm use perl wrapper instead of 'perl.real' directly
+        $name =~ s/perl\.real/perl/ if ($name =~ /perl\.real/);
+
         my ($abs, $use_dir);
         if ($self->file_name_is_absolute($name)) {     # /foo/bar
             $abs = $name;
@@ -2097,6 +2100,7 @@ sub init_PERL {
 
     $self->{PERL} ||=
         $self->find_perl(5.0, \@perls, \@defpath, $Verbose );
+    
 
     my $perl = $self->{PERL};
     $perl =~ s/^"//;
